### ISSUE TITLE
Should Mayor just get two votes?

### ISSUE THREAD

**Issue by Anusien** [Anusien/Town-Of-Us#82]
There are a lot of problems with the current implementation of Mayor. Maybe it would work better to get two votes.

**Comment by k-electron** [comment]
yes - that makes a lot more sense. the current bugs around the sound and vote counting makes it pretty annoying.

early game the mayor doesnt have too many votes banked and late game they dont need more than a couple of votes so the current implementation is needlessly complicated.
